ingest_api/infrastructure/construct.py

```diff
@@ -296,7 +293,7 @@ def build_ingestor(
         db_secret: secretsmanager.ISecret,
         db_vpc: ec2.IVpc,
         db_security_group: ec2.ISecurityGroup,
-        db_subnet_public: bool,
```

Comment:
this is no longer required since we already account for `db_publicly_accessible` from the database when selecting the subnets